app_feup/lib/view/Pages/login_page_view.dart

```diff
@@ -153,6 +155,33 @@ class _LoginPageViewState extends State<LoginPageView> {
     );
   }
 
+  /// Creates the widget for the user to choose their faculty
+  Widget createFacultyInput() {
+    final faculties = ['faup', 'fbaup', 'fcup', 'fcnaup', 'fadeup', 'fdup',
+      'fep', 'feup', 'ffup', 'flup', 'fmup', 'fmdup', 'fpceup', 'icbas'];
+
+    return DropdownButton<String> (
+      value: faculty,
+      items: faculties.map<DropdownMenuItem<String>>((String value) {
+        return DropdownMenuItem<String> (
+          value: value,
+          child: Text(value),
+        );
+      }).toList(),
+      onChanged: (newDropdownValue) {
+        setState(() {
+          faculty = newDropdownValue;
+        });
+      },
+      isExpanded: true,
+      dropdownColor: Color.fromARGB(255, 0x75, 0x17, 0x1e),
```

Comment:
Colors that are from the theme can be obtained through the `context` object: https://api.flutter.dev/flutter/widgets/State/context.html

This color could then be obtained through this call:
```suggestion
      dropdownColor: Theme.of(context).accentColor,
```

Can you do the same for all the other colors used? TextStyles can also be obtained in the same, check the file `theme.dart`